tests/conftest.py

```
@@ -82,61 +84,49 @@ def zeromq_broker(tmp_path):
 
-    :param broker: A ``ZeromqBroker`` instance (has ``service_dir``, ``is_running``, etc.)
-    """
-    broker.service_dir.mkdir(parents=True, exist_ok=True)
-
-    if broker.is_running:
-        return
+    zeromq_broker.service_dir.mkdir(parents=True, exist_ok=True)
 
-    subprocess.Popen(
-        [sys.executable, '-m', 'aiida.brokers.zeromq.service', '--service-dir', str(broker.service_dir)],
+    process = subprocess.Popen(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)